project1/application.py

```diff
@@ -1,29 +1,92 @@
+from models import User
 import os
-
-from flask import Flask, session, render_template
+import datetime
+from flask import Flask, session, render_template, request
 from flask_session import Session
 from sqlalchemy import create_engine
 from sqlalchemy.orm import scoped_session, sessionmaker
+from flask_sqlalchemy import SQLAlchemy
+# from models import User
+
+
+
 
 app = Flask(__name__)
 
 # Check for environment variable
 if not os.getenv("DATABASE_URL"):
     raise RuntimeError("DATABASE_URL is not set")
 
+
 # Configure session to use filesystem
+app.config["SQLALCHEMY_DATABASE_URI"] = "postgres://yainmriqptbdia:ef9eb65b36e26462fab4fa3190ea7982f65a5b450e7be9789b4ac2f507b8f634@ec2-54-81-37-115.compute-1.amazonaws.com:5432/d226orf1s2l1ac"
```

Comment:
Don't display the url,it needs e hidden and given in cmd due to security reasons

Reply:
Removed ma'am.  committed new change